pkg/policy/api/entity.go

```diff
@@ -109,6 +115,7 @@ func InitEntities(clusterName string, treatRemoteNodeAsHost bool) {
 		endpointSelectorInit,
 		endpointSelectorHealth,
 		endpointSelectorUnmanaged,
+		endpointSelectorKubeAPIServer,
```

Comment:
I think that we should either consistently define `cluster` to include the kube-apiserver, or consistently define it to not include kube-apiserver. Given that it's the management plane for the cluster, it seems reasonable to include it here even if it's not hosted on nodes that are traditionally considered part of the k8s cluster.

Reply:
That makes sense to me. Just needs to be documented clearly as it may be confusing to other users. I didn't see any docs changes in this PR so I'm guessing they are coming as a follow up PR?

Reply:
agree. I've added a task to the PR description to make sure we either address it here or carry it on to a subsequent follow-up PR.

Reply:
@joestringer Did you add the task to the description and then remove it because it wasn't needed, or do we still need to handle this?

Reply:
Oops, sorry I see it now. Didn't see it the first time because I was looking at the first few works on the bullet points.